ETR01SDK-630: Docs: refactor CMake options mentions in tutorials (#537)
tropicsquare/libtropic: docs/tutorials/linux/usb_devkit/identify_chip.md

    After this, you should see an output in your terminal.

## Configuration

> Besides the [Libtropic CMake options](../../../reference/integrating_libtropic/how_to_configure/index.md) used to configure Libtropic, there is the following CMake option specific to this example:

Reviewer: ```suggestion
In addition to the [Libtropic CMake options](../../../reference/integrating_libtropic/how_to_configure/index.md) used to configure Libtropic, this example provides the following CMake option:
```

> - `LT_USB_DEVKIT_PATH` (default: `"/dev/ttyACM0"`) to set the path to the USB device representing the USB DevKit serial port:

Reviewer: ```suggestion
- `LT_USB_DEVKIT_PATH` (default: `"/dev/ttyACM0"`) sets the path to the USB device representing the USB DevKit serial port:
```
